pennylane/transforms/qcut.py

```diff
@@ -2725,14 +2878,170 @@ def find_and_place_cuts(
          b: ──RY(0.4)──────╰X──│───╰X────────────RY(0.6)──╰┤ ⟨Y ⊗ Z⟩
          1: ──PrepareNode──────╰C───MeasureNode────────────┤
 
+    Alternatively, if all we want to do is to find the optimal way to fit a circuit onto a smaller
+    device, a :class:`~.CutStrategy` can be used to populate the necessary explorations of cutting
+    parameters. As an extreme example, if the only device at our desposal is a 2-qubit device, a
+    simple cut strategy is to simply specify the the ``max_free_wires`` argument (or equivalently
+    directly passing a :class:`~.Device` to the ``device`` argument):
+
+    >>> cut_strategy = qml.transforms.qcut.CutStrategy(max_free_wires=2)
+    >>> print(cut_strategy.get_cut_kwargs(graph))
+     [{'num_fragments': 2, 'imbalance': 0.5714285714285714},
+      {'num_fragments': 3, 'imbalance': 1.4},
+      {'num_fragments': 4, 'imbalance': 1.75},
+      {'num_fragments': 5, 'imbalance': 2.3333333333333335},
+      {'num_fragments': 6, 'imbalance': 2.0},
+      {'num_fragments': 7, 'imbalance': 3.0},
+      {'num_fragments': 8, 'imbalance': 2.5},
+      {'num_fragments': 9, 'imbalance': 2.0},
+      {'num_fragments': 10, 'imbalance': 1.5},
+      {'num_fragments': 11, 'imbalance': 1.0},
+      {'num_fragments': 12, 'imbalance': 0.5},
+      {'num_fragments': 13, 'imbalance': 0.05},
+      {'num_fragments': 14, 'imbalance': 0.1}]
+
+    The printed list above shows all the possible cutting configurations one can attempt to perform
+    in order to search for the optimal cut. This is done by directly passing a
+    :class:`~.CutStrategy` to :func:`~.find_and_place_cuts`:
+
+    >>> cut_graph = qml.transforms.qcut.find_and_place_cuts(
+            graph=graph,
+            cut_strategy=cut_strategy,
+        )
+    >>> print(qml.transforms.qcut.graph_to_tape(cut_graph).draw())
+     0: ──RX──//─╭C──//────────╭C──//─────────┤ ╭<X@Y@Z>
```

Comment:
I vote for preventing it for both auto and manual placement.

Reply:
Actually, there could be a way to directly prevent these kind of cuts to be placed by kahypar: I could preprocess the circuit and place fake manual cuts before all measurements to force kahypar to not cut them. But this will involve changing the logic in a few other places... best not touch it this PR 😅